autogen/agentchat/contrib/capabilities/teachability.py

```diff
@@ -23,6 +23,12 @@ class Teachability(AgentCapability):
     To make any conversable agent teachable, instantiate both the agent and the Teachability class,
     then pass the agent to teachability.add_to_agent(agent).
     Note that teachable agents in a group chat must be given unique path_to_db_dir values.
+
+    When adding Teachability to an agent, the following are modified:
+    - The agent's system message is appended with a note about the agent's new ability.
+    - A hook is added to the agent's `process_last_received_message` hookable method,
+    and the hook potentially modifies the last of the received messages to include earlier teachings related to the message.
+    Not modifying the stored message history. Stores apparent teachings in new memos.
```

Comment:
"Added teachings do not propagate into the stored message history. If new user teachings are detected, they are added to new memos in the vector database."

Reply:
Thanks. Modified.